core/vm/jump_table.go

```diff
@@ -42,9 +42,6 @@ type operation struct {
 
 	// memorySize returns the memory size required for the operation
 	memorySize memorySizeFunc
-
-	// undefined denotes if the instruction is not officially defined in the jump table
-	undefined bool
```

Comment:
This was added in #30418 and Martin wasn't the happiest about it. IIRC it was used for EOF, so should be okay to remove.